docs/hardware_sets.md

```diff
@@ -0,0 +1,50 @@
+# Hardware sets
+
+Hardware sets are collections of hardware device types along with their associated
+parameters. They are used in FINESSE to represent a particular hardware rig (e.g.
+FINESSE or UNIRAS), so that users can easily swap between them. FINESSE comes with some
+built in configurations, but users can also define their own, either for convenience
+during testing or to build on FINESSE's functionality.
+
+Hardware sets are represented in a [YAML](https://yaml.org) format. Custom hardware sets
+can be created and imported into FINESSE. Here is an example:
+
+```yaml
+name: My hardware set
+devices:
+  stepper_motor:
+    class_name: stepper_motor.st10_controller.ST10Controller
+    params:
+      port: "0403:6011 FT1NMSVR (4)"
```

Comment:
Good point :laughing:. You *could* work out these values yourself, but it would be a pain. The easiest way to do it is to open FINESSE, go to the manual device management panel and the port strings for the different USB serial devices will be listed there.

Should I add a note to that effect?

Reply:
That might be useful, yes :)